curbs/README.md

```diff
@@ -68,11 +69,9 @@ All endpoints return a JSON object containing the fields as specified in the [RE
 
 Endpoint: `/curbs/zone`  
```

Comment:
Yes I think you are right, some CDS endpoints should be plural, since it makes sense, is good API policy, and aligns with MDS. The original code contribution had them singular. 

I wish we would have seen this since it was added back in June. Very close to launch now, but I'll try to work it in and  post the results here.

Reply:
Updated the spec across APIs and examples to make all endpoints plural (since all return lists of objects). /events/status still is /status since 'status' is acceptable as a plural and preferred (according to some sources). 

Take a look here and let me know if I missed anything.

https://github.com/openmobilityfoundation/curb-data-specification/tree/feature-release-work-1/curbs#query-curb-zones